ast/ast.go

```diff
@@ -1445,6 +1448,115 @@ type CastNumValue struct {
 //
 // ================================================================================
 
+// Options is generic OPTIONS clause node without key and value checking.
+//
+//	OPTIONS ({{.Records | sqlJoin ","}})
+type Options struct {
+	// pos = Options
+	// end = Rparen + 1
+
+	Options token.Pos // position of "OPTIONS" keyword
+	Rparen  token.Pos // position of ")"
+
+	Records []*OptionsDef // len(Records) > 0
+}
+
+// Field finds name in Records, and return its value as Expr.
+// The second return value indicates that the name was found in Records.
+// It is not exposed because you can use *Field methods.
```

Comment:
Is it true?

Reply:
Oh, I forgot to fix comment.
It is not true. I will fix it.

Reply:
Fixed https://github.com/cloudspannerecosystem/memefish/pull/142/commits/b469bb938f7e51284a572b45f0895f909079c79d